### ISSUE TITLE
Measure Manager sometimes updates measure.xml in local BCL, preventing update measure from working

### ISSUE THREAD

**Issue by DavidGoldwasser** [NREL/OpenStudio#3406]
The <version_modified> tag can get updated with a date newer than what is on the remote BCL version of the measure. This prevents the user from updating the measure. I'm not sure what is causing this, but has happened to at least two users who where not developing measure. The distinction here, is that I can see a way this might happen to measure developer. If you set your "My Measures" directory to be your local BCL, it may update the XML in a number of situations. But these two users did not do this.

It may be more extensive of an issue, the two we became aware of were when an old measure failed to run on a newer version of OpenStudio because of non-backwards compatible `openstudio-standards` gem. This could happen in more cases, and users would not see an issue, they just would not be made aware in the GUI that a newer measure is available and if we make them aware of it, they still can't download it. This was seen in OS app, but would also impact PAT. While the ideal goal is to prevent this from happening, we could have a logic that assumes BCL is always the most current, and will always offer to update when <version> tags don't match, without worrying about what <version_modified> says. Since nobody should every edit measure in BCL directory, I don't see a down side to this. You should never have anything more current in your BCL directory than is online at BCL.

The work around with current installer is to delete or remove their BCL directory, quit OpenStudio, and then re-download the measures they need.

**Comment by DavidGoldwasser** [comment]
@macumber as an odd example, here is the diff log in my current checkout of develop branch of OpenStudio-measure.

What triggered the change to the measure.xml?

- Maybe the readme.md got deleted, then measure manager updated the measure(without that file) giving it a new version id and date, and then after the readme.md came back, the measure manager again ran and updated the version id and date.
- Another thing I have seen. Somehow, the measure fails to load and produces an error message in the measure.xml (maybe I inadvertently run it with older version of OpenStudio). Then when I run it again the result is new version id and updated date from what I intended.
- As to how this might happen in BCL directory where nobody runs measure tests. We did have a 2.x.0 release that had issues with standards. I can't remember if it was 2.5. or 2.6? At any rate, when the measure fails to load (I guess it would have to fail arguments section not run section?) the measure.xml might be updated to show the error. At this point when they change their installer and get the measure working, that measure now looks like it is newer than what is on BCL.

There was a time that we didn't even have a date, and I thought the logic was that if the version ID on BCL isnt' the same as what is in BCL directory, it was assumed to be an update, as we had no way to know which was newer or older. That logic may have actually been better than trusting the date.

But this still isn't ideal. User will get an alert from time to time about an update to the measure, when in reality, there is no update, it is just a tweaked measure.xml.

![screen shot 2019-01-18 at 2 43 37 pm](https://user-images.githubusercontent.com/4857055/51414482-ea4f7700-1b2f-11e9-8cf0-5fde6b4137ff.png)
